tapeagents/llms.py

```diff
@@ -1289,8 +1291,12 @@ def trainable_llm_make_training_text(prompt: Prompt, output: LLMOutput, tokenize
     Returns:
         TrainingText: A dataclass containing:
 
-            - text (str): The formatted conversation text
-            - n_predicted (int): Length of the output text portion
+            - text (str): The formatted conversation text (prompt + output)
+            - n_predicted (int): Length of the output tokens
```

Comment:
let's rename it to `n_predicted_tokens` to avoid any misunderstanding in the future.